src/components/card/demoBasicUsage/index.html

```diff
@@ -1,7 +1,7 @@
 <div ng-controller="AppCtrl" ng-cloak >
   <md-content class="md-padding" layout-xs="column" layout="row">
     <div flex-xs flex-gt-xs="50" layout="column">
-      <md-card>
+      <md-card md-theme="dark">
```

Comment:
I'm not sure, it looks a bit weird in the demos, when the first card is dark. In my opinion it would be awesome if we have a checkbox for toggle the dark theme.